src/api/spec/controllers/webui/user_controller_spec.rb

```diff
@@ -6,7 +6,6 @@
   let!(:admin_user) { create(:admin_user, login: "king") }
   let(:deleted_user) { create(:deleted_user) }
 
-  it { is_expected.to use_before_action(:require_login) }
```

Comment:
require_login and require_admin are two different before filters... they are defined in the user controller... I wanted to test it, what's wrong?